### PR TITLE
Add information for rust-bitcode

### LINKED ISSUE #4: iOS dependencies
Hey!

Awesome project! I'm trying to run the iOS app (haven't tried Android yet) and I'm having some issues. Is there anything not in the README that's needed?

I'm running on stable.

When I run the iOS app in XCode I get

```
+ RUSTFLAGS='-Z embed-bitcode'
+ cargo +ios-arm64 build --target aarch64-apple-ios --release --lib
error: no such subcommand: `+ios-arm64`
+ cargo build --target=x86_64-apple-ios --release
   Compiling log v0.4.8
   Compiling libc v0.2.60
   Compiling core-foundation-sys v0.6.2
   Compiling cfg-if v0.1.9
error[E0463]: can't find crate for `core`
  |
  = note: the `x86_64-apple-ios` target may not be installed

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: could not compile `cfg-if`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: linking with `cc` failed: exit code: 1
```

I have installed `x86_64-apple-ios` and `aarch64-apple-ios`.

### PR DESCRIPTION
**Pull request by ineiti**
The installation instructions were missing for the iOS part.
Also some of the import parts in the xcode project were wrong.

Closes #4

### CODE REVIEW

File: ios_app/build-rust-xcode.sh [hunk 1]
```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 
 # Ensure that `cargo` is in PATH, using the default location.
-export PATH="$PATH:${HOME}/.cargo/bin"
+. "$HOME/.cargo/env"
```
Comment (i-schuetz) [inline ios_app/build-rust-xcode.sh:4]: 👍

File: README.md [hunk 2]
```diff
@@ -106,12 +106,26 @@ rustup target add aarch64-linux-android armv7-linux-androideabi x86_64-linux-and
 
 ## iOS specific steps
 
-- Add targets
+- Install rust 1.46.0 with MacOSX binary support
```
Comment (i-schuetz) [inline README.md:109]: macOS :)

**Comment by i-schuetz** [comment]
It works, thanks 👍! can we can do something to not have to specify the version in the app's script (and ideally use a placeholder in the instructions too)?

**Comment by ineiti** [comment]
Did you test it on a clean repo? I had to compile it manually once for all the rust libraries, and then let Xcode do the compiling of the main rust code and the linking.

With regard to multiple versions, this might be a bit tricky. The `~/.rustup/toolchains` can hold multiple directories. 
- should it just chose the first that starts with `ios-arm64`? 
- Or complain if there are multiple `ios-arm64`, but take the one available if there is only one?

**Comment by i-schuetz** [comment]
Hmm I had to run 
```sh
cargo +ios-arm64-1.46.0 build --target aarch64-apple-ios --release --lib
cargo build --target=x86_64-apple-ios --release
```
directly on the terminal once.. it seems that there are linker issues when the script is executed from Xcode. 
> "note: ld: library not found for -lSystem "

Maybe related? https://github.com/TimNN/cargo-lipo/issues/41

Let's skip the version thing for now, rust-bitcode's last stable release is from about 6 months ago, so not that much of an issue.

**Comment by i-schuetz** [comment]
Thanks for your contribution!

**Pull request merged**